src/async.ts

```diff
@@ -1,187 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
This was used for the previous linter, not sure of it's exact role but it seems easier to keep all the components running on one process?